### DISCUSSION TITLE
Reason for suffix on history completion?

### DISCUSSION THREAD

**Discussion by jcassee** [marlonrichert/zsh-autocomplete#324] [Q&A] [answered]
Hi, since a114e5324abffb1a279582c3ab20ebe2a696f984, the history suffix (semicolon) is removed for most selections. What is the reason for showing it during completion? I'm sure I can get used to it, but at the moment it mostly confuses me. Would it be an idea to just add it when using multiselect?

**Comment by marlonrichert** [comment, marked as answer]
The `;` suffix is necessary to make multiselect usable. The way this works in Zsh, is that each completion can have a suffix that can be removed afterwards. This is the only way we can insert `;` to separate multiple history lines. The suffix is automatically removed (under most circumstances) for the last line you insert. It’s not possible to insert separators after the fact.

**Reply by jcassee** [reply]
Alright, thanks for the explanation!
